### ISSUE TITLE
GraphQL interfaces are not generated if mappingConfig has "generateApis = false"

### ISSUE THREAD

**Issue by PavelSusloparov** [kobylynskyi/graphql-java-codegen#9]
**Is your feature request related to a problem? Please describe.**
Interfaces generation is not supported as of now. 
schema.graphqls
```
interface MyInterface {
  id: String
  ...
}
```
There is no class MyInterface automatically generated in the target folder.

**Describe the solution you'd like**
The interface is automatically generated in the target folder.
MyInterface.java
```
public interface MyInterface {

	public String getId();
	public void setId(String id);
}
```

**Comment by kobylynskyi** [comment]
@PavelSusloparov, interfaces generation is already supported since 1.0.
You can see an example here:
* GraphQL interface: https://github.com/kobylynskyi/graphql-java-codegen/blob/master/src/test/resources/schemas/github.graphqls#L4111
* Java class: https://github.com/kobylynskyi/graphql-java-codegen/blob/master/src/test/resources/expected-classes/ProfileOwner.java.txt

**Comment by kobylynskyi** [comment]
@PavelSusloparov, now I got you.
interfaces are not generated when you are passing a flag `generateApis = false`
I will fix it in the next release.

**Comment by PavelSusloparov** [comment]
Ah, I see. For now, I will remove `generateApis = false` from the configuration.
Thanks!

**Comment by kobylynskyi** [comment]
Fixed and released in [1.2.2](https://github.com/kobylynskyi/graphql-java-codegen/releases/tag/v1.2.2)
Will release maven and gradle plugins shortly.
